hibike/hibike_communicator.py

```diff
@@ -61,24 +61,25 @@ def process_output(self):
    
          #If it is a device value, cache it in the dictionary.
          if command == "device_values":
-             try:
-                uid = output[1][0].keys()[0]
-             except TypeError:
-                print("Dict Keys TypeError")
-                continue
-             print("UID: ", uid, "\n")
-             try:
+            uid = 0
+            for key in output[1][0].keys():
+                uid = key
+                #print ("Key is", uid, "this is key")
+                #break
+                
+            #print("UID: ", uid, "\n")
+            try:
                 params_values = output[1][0][uid]
-             except IndexError:
-                print("No UIDs detected for a device_values request")
+            except KeyError:
+                #print("No UIDs detected for a device_values request")
                 params_values = [{}]
```

Comment:
You can probably condense this entire section to `params_values = next(output[1][0].values(), [{}])`.